src/Vtiger.php

```diff
@@ -618,8 +578,7 @@ protected function _processResponseError($processedData)
         if (!isset($processedData->error)) {
             throw VtigerError::init($this->vTigerErrors, 3);
         }
-
+        dd($processedData);

```

Comment:
Dont think this should of been left in lol

Reply:
Whoops, seems I left that in there from when I was debugging! Nice catch haha